core/persist/viewmanager/ViewManagerModel.ts

```diff
@@ -36,24 +34,36 @@ export interface ViewManagerConfig {
     /** True (default) to allow user to mark views as favorites. Requires `persistWith`. */
     enableFavorites?: boolean;
     /**
-     * True to allow the user to publish or edit globally shared views. Apps are expected to
+     * True to allow the user to publish or edit the global views. Apps are expected to
      * commonly set this based on user roles - e.g. `XH.getUser().hasRole('MANAGE_GRID_VIEWS')`.
      */
-    enableSharing?: Thunkable<boolean>;
-    /** Used to persist the user's last selected + favorite views and autoSave preference. */
-    persistWith?: PersistOptions;
+    manageGlobal?: Thunkable<boolean>;
```

Comment:
This reads as a bit too terse for me - just barely, but would be nice for it to be as clear as possible.  Could we do `manageGlobalViews`?  I know that "views" could be considered implied - it's a view manager - but still...

Would really like `enableGlobalViewManagement` but even I admit that's pretty long.  An earlier iteration of project used `canManageGlobal` - not sure if the `can` prefix is something we use anywhere else.

Alternatively, any sense we want an object here like `enableSharing: {global: boolean}`?  If we were more sure of our next steps with per-user sharing maybe we would know.

Again, don't need to change, this just stuck out at me.